mirrord/agent/src/main.rs

```diff
@@ -152,6 +157,7 @@ struct ClientConnectionHandler {
     tcp_outgoing_api: TcpOutgoingApi,
     udp_outgoing_api: UdpOutgoingApi,
     dns_sender: Sender<DnsRequest>,
+    send_recv_manager: SendRecvManager,
```

Comment:
why is it SendRecvManager?

Reply:
since it will manage both send/recv, i thought it would make sense to have a common interface for this? would you rather just have a Sender?

Reply:
if it's recv then it would go through our normal udp socket flow